exchange/src/exchange/traders/trader_types/generic_trader.hpp

```diff
@@ -15,7 +16,7 @@ class GenericTrader {
     const std::string USER_ID;
     const double INITIAL_CAPITAL;
     double capital_delta_{};
-    std::unordered_map<util::Ticker, double> holdings_{};
+    emhash7::HashMap<util::Ticker, double, absl::Hash<util::Ticker>> holdings_{};
```

Comment:
Why aren't you using the `TickerMap` typedef?

Reply:
TickerMap is a different type (ticker->ticker_info, which deserves a rename but w/e for now)